janus_core/processing/post_process.py

```diff
@@ -247,12 +248,32 @@ def compute_vaf(
     n_steps = len(momenta)
     n_atoms = len(momenta[0])
 
-    # If filter_atoms not specified use all atoms
-    filter_atoms = [
-        atom if atom[0] is not None else range(n_atoms) for atom in filter_atoms
-    ]
+    filtered_atoms = []
+    atom_symbols = data[0].get_chemical_symbols()
+    symbols = set(atom_symbols)
+    for atoms in atoms_filter:
+        if any(atom is None for atom in atoms):
+            # If atoms_filter not specified use all atoms.
+            filtered_atoms.append(range(n_atoms))
+        elif all(isinstance(a, str) for a in atoms):
+            # If all symbols, get the matching indices.
+            atoms = set(atoms)
+            if atoms.difference(symbols):
+                raise ValueError(
+                    f"{atoms.difference(symbols)} not allowed in VAF"
+                    f", allowed symbols are {symbols}"
+                )
+            filtered_atoms.append(
+                [i for i in range(len(atom_symbols)) if atom_symbols[i] in list(atoms)]
+            )
+        elif all(isinstance(a, int) for a in atoms):
+            filtered_atoms.append(atoms)
+        else:
+            raise ValueError(
+                "Cannot mix element symbols and indices in vaf atoms_filter"
+            )
 
-    used_atoms = {atom for atoms in filter_atoms for atom in atoms}
+    used_atoms = {atom for atoms in filtered_atoms for atom in atoms}
```

Comment:
Hasn't `filter_atoms` been replaced by `atoms_filter` generally? Is this still just renaming, or actually changing something? (also below)

Reply:
I don't really mind in terms of implementation (this seems reasonable), just checking it's intentional